py3status/modules/arch_updates.py

```diff
@@ -94,6 +96,13 @@ def _get_cower_updates(self):
         except self.py3.CommandError as ce:
             return len(ce.output.splitlines())
 
+    def _get_trizen_updates(self):
+        try:
+            updates = self.py3.command_output(["trizen", "-Sua"])
```

Comment:
Please check if we want `Sua` or `Qua` optionally with `q` (quiet). 
1) https://github.com/trizen/trizen/issues/160
2) Executing `trizen -Qua` gave me an interactive prompt.
3) For 1 update, `Qua` gave me 1. `Sua` gave 4 (due to prompt lines).

Reply:
The prompt is only displayed in interactive shells (https://github.com/trizen/trizen/blob/master/trizen#L2570-L2580) so that shouldn't be a problem. From a quick look at the code, it actually looks like both commands call the same function, but `-Q` (query) is probably a more logical choice (I used `-Sua` because of trizen/trizen#33). `-q` disables version number output which we don't need so that should fine to use.

Reply:
```
 _______
< -Quaq >
 -------
        \
         \
          🦆
```
`-Quaq` is also much more fun than `-Sua`.

Reply:
>From a quick look at the code, it actually looks like both commands call the same function

Sorry. I think 4 came from me forgetting to override aur with `trizen` when switching branches.

>but -Q (query) is probably a more logical choice (I used -Sua as recommended by trizen/trizen#33). -q disables version number output which we don't need so that should fine to use.

Agreed. `-Quaq` looks nice and is recent than `-Sua` (2017 vs 2018... by 5~ months or so).

>-Quaq is also much more fun than -Sua.

:+1: